packages/quip-cli/src/commands/login.ts

```diff
@@ -119,7 +125,11 @@ export const login = async ({
             } - response: ${JSON.stringify(tokenResponse, null, 2)}`
         );
     }
-    await writeSiteConfig(config, site, { accessToken });
+    if (displayTokenOnly) {
+        log(chalk`{magenta Your access token is "${accessToken}".}`);
```

Comment:
since `--export` is for an CI/CD use cases i think logging out the access token without "your access token is" would be easier for following flows/actions to consume.

Reply:
My understanding is that the developers need to copy/past the token from the terminal eventually. As when `quip-cli  login --export` is executed, the login url will also be printed in the terminal (the terminal stdout is "polluted"). 

![image](https://user-images.githubusercontent.com/2281521/141376866-0f37402f-59d2-498e-aa47-5a9d7ad7a2ce.png)

Reply:
oh you are right

Reply:
Thank you.